src/retractions.jl

```diff
@@ -934,3 +934,30 @@ function retract_sasaki! end
 
 Base.show(io::IO, ::CayleyRetraction) = print(io, "CayleyRetraction()")
 Base.show(io::IO, ::PadeRetraction{m}) where {m} = print(io, "PadeRetraction($m)")
+
+#
+# default estimation methods pass down with and without the point type
+function default_estimation_method(M::AbstractManifold, ::typeof(inverse_retract))
+    return default_inverse_retraction_method(M)
+end
+function default_estimation_method(M::AbstractManifold, ::typeof(inverse_retract), T)
+    return default_inverse_retraction_method(M, T)
+end
+function default_estimation_method(M::AbstractManifold, ::typeof(inverse_retract!))
```

Comment:
I think we can only define these methods for non-mutating functions for simplicity?

Reply:
Sure, sounds fair enough and makes testing a bit shorter ;)